src/pages/contract/verify/index.tsx

```diff
@@ -14,10 +14,36 @@ const sleep = (ms: number) =>
     setTimeout(r, ms);
   });
 
+const initialLibraryValues: { [key: string]: string } = {
+  library1Name: "",
```

Comment:
Are we okay with displaying all 10 inputs at once? Or should we make it a bit more dynamic?

_(blockscout)_
![image](https://github.com/BirthdayResearch/metascan/assets/17063281/704ed499-0f9e-4c77-8a98-2fa202792ea6)

Reply:
I am following etherscan as we don't have any screen in figma
<img width="477" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/53080940/f8747a4a-40ec-4ec1-8a93-e69acec3e621">